src/snapred/backend/dao/indexing/Versioning.py

```diff
@@ -1,79 +1,51 @@
-from typing import Any, Optional
-
-from numpy import integer
-from pydantic import BaseModel, computed_field, field_serializer
+from pydantic import BaseModel, ConfigDict, field_validator
 
 from snapred.meta.Config import Config
+from snapred.meta.Enum import StrEnum
 
 VERSION_START = Config["version.start"]
 VERSION_NONE_NAME = Config["version.friendlyName.error"]
```

Comment:
I'm wondering if we should put type annotations on these constants?  For example, I'm assuming that `VERSION_START: int` and `VERSION_NONE_NAME: str` and etc.